/* -*- Mode:protobuf; indent-tabs-mode:nil; -*- */
// More information: https://developers.google.com/protocol-buffers/docs/cpptutorial

syntax = "proto3";

package ndn.ndvr.proto;


message DvInfoIBF {
  message NextHop {
    uint64 number = 1;
  }

  message Entry {
    string prefix = 1;
    uint64 seq = 2;
    string originator = 3;
    NextHop next_hops = 4;
  }

  repeated Entry entry = 1;
}
